examples/single/bundle/src/App.js

```diff
@@ -24,15 +24,15 @@ import "regenerator-runtime/runtime";
 
 import {
   Session,
-  getAuthFetcherWithDependencies
+  getClientAuthnWithDependencies
```

Comment:
I don't think we should be using `Authn` in our function names - i.e. this should be `getClientAuthenticationWithDependencies()`. (But we can change in another PR - ticket: https://inrupt.atlassian.net/browse/SDK-1181).